src/licence-status-banner/index.jsx

```diff
@@ -48,13 +49,20 @@ function LicenceStatusBanner({ licence, licenceType, isPdf, dateFormat, colour,
           <a href="#">{open ? 'Show less' : 'Show more'}</a>
         </p>
         <p className="status">
-          { title || <Snippet>{`invalidLicence.status.${licenceStatus}`}</Snippet> }
+          { title || <Snippet>{`invalidLicence.status.${establishmentSuspended ? 'establishmentSuspended' : licenceStatus}`}</Snippet> }
```

Comment:
It looks like this will show the establishment suspended content on a licence which is otherwise inactive (i.e. revoked, expired or suspended itself). I would probably only show the establishment suspended message if the licence would otherwise be active.